FitNesseRoot/FitNesse/SuiteAcceptanceTests/SuiteSlimTests/SlimSymbols/NamingConvention/content.txt

```diff
@@ -0,0 +1,31 @@
+|import            |
+|fitnesse.slim.test|
+
+!3 Slim symbols names must start with a letter, followed by letters, digits and underscores
+
+!|test slim          |
+|string|getStringArg?|
+|foo   |$x3_=        |
+|$x3_  |foo          |
+
+!3 Referencing undefined Slim symbols names raise no erro but return the symbol name
+
+!|test slim              |
+|string    |getStringArg?|
+|$UNDEFINED|$UNDEFINED   |
+
+!3 Slim symbols names are case sesitive
+
+!|test slim          |
+|string|getStringArg?|
+|lower |$x=          |
+|upper |$X=          |
+|$x    |lower        |
+|$X    |upper        |
+
+!3 Slim symbols names also support UTF8 names
+
+!|test slim   |3            |
+|string       |getStringArg?|
+|ØÅÆØÅÆ_ØÅÆØÅÆ|$Æ_ØÅ=       |
+|$Æ_ØÅ        |ØÅÆØÅÆ_ØÅÆØÅÆ|
```

Comment:
Hi @hcf and @amolenaar

I added this test page with this UTF8 test: FitNesseRoot/FitNesse/SuiteAcceptanceTests/SuiteSlimTests/SlimSymbols/NamingConvention

Think this should cover the required testing
and in addition provides the documentation to the community that this new feature has been added.

I am a big fan of "Eat your own dog food" and prefer adding wiki tests then Junit tests.
And I think this is a nice example that wiki tests can be more efficient than Junit as it takes less lines to express the feature in a wiki page than in Junit.

If you see any feature not covered than let me know and I can add it.